samples/rag/csharp-ooproc/EmailPromptDemo.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
rag-aisearch also uses SemanticSearch with AISearch, please verify that as well.

Reply:
done!